src/config.ts

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config();
+
+export const mySQLConfig = {
+    host: process.env.MYSQL_HOST,
+    port: process.env.MYSQL_PORT,
+    user: process.env.MYSQL_USER,
+    rootPassword: process.env.MYSQL_ROOT_PASSWORD,
```

Comment:
have you checked the application running?

I believe, should be `password`, as in previous.

Also, worth to change in env variable as well to `MYSQ_PASSWORD `

Reply:
@melnychukvitaliy We are not use `MYSQL_ROOT_PASSWORD`, I put it into `.env` for setting of mysql password for docker-compose [Link](https://hub.docker.com/_/mysql). I think we can move it back to docker-compose and use `environment` instead of `env_file` for configure docker

Reply:
@kanoru3101 You can make modification in docker-compose and remove `dotenv` library and `.env` file.